src/utils/ApiClient.js

```diff
@@ -258,15 +258,20 @@ export default class ApiClient {
      */
     wrapCallback(httpMethod, callback = () => null) {
         return (err, res) => {
-            const expectedContentType = (this.isEncrypted) ? "application/jose+json" : "application/json";
-            const invalidContentType = res && res.header && res.status !== 204 && res.header["content-type"].indexOf(expectedContentType) === -1;
-            if (invalidContentType) {
-                callback([{
-                    message: "Invalid Content-Type specified in Response Header",
-                }], res ? res.body : undefined, res);
-                return;
+            const contentTypeHeader = res && res.header ? res.header["content-type"] : undefined;
+            if (!err) {
+                const expectedContentType = (this.isEncrypted) ? "application/jose+json" : "application/json";
+                const invalidContentType = res && res.status !== 204
+                  && contentTypeHeader && contentTypeHeader.indexOf(expectedContentType) === -1;
```

Comment:
is `contentTypeHeader` String? if it is, why not use [String.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) ?